### DISCUSSION TITLE
Is there any way to adjust resolution and max distance for each light?

### DISCUSSION THREAD

**Discussion by POPREQ** [bnpr/Malt#282] [Q&A] [answered]
I would like to separate the light groups by character and background and set the higher resolution lights only on the character.

Can I do that in malt so that eevee has a max distance adjustment item for each light?

**Comment by pragma37** [comment, marked as answer]
This was not possible but I just implemented it in the Development version (12b5739d19c8d6dce9051951a6e62371e2e1ea13) 👍 
You have to enable the *Override Global Settings* option.

![imagen](https://user-images.githubusercontent.com/36610999/162537454-e594ebe8-60a1-41b9-8b1c-e91902b8a60f.png)

**Reply by POPREQ** [reply]
I tried it immediately. It's brilliant and so easy to use!
